server/src/message/types.ts

```diff
@@ -0,0 +1,19 @@
+import { RainIntesity } from '../weather/types';
+
+export interface WeatherMessage {
+  id: string;
+  message: string;
+  rain_intensity: RainIntesity;
+}
+
+export interface WeatherMessageFilter {
+  rain_intesity?: RainIntesity;
+}
+
+export const WhenWillRainText = {
+  afternoon: 'hoje à tarde',
+  night: 'hoje à noite',
+  tomorrow: 'amanha',
```

Comment:
I think morning would be better than tomorrow, since we expect the user to check the app daily.